app/assets/translations/appConstants/bebbo/en.ts

```diff
@@ -442,17 +442,17 @@ const en = {
                 "restOfTheWorldAlertTitle": "Attention",
                 "restOfTheWorldOkTitle": "OK",
                 "restOfTheWorldAlertText": "By selecting Rest of the world, you will enjoy all features of Bebbo, but note that vaccinations and health check-ups are not customized to a specific national health program.",
-                "hcDefinedReminderText": "Enter the date and time of the reminder",
-                "reminderalertTextDefined": "Reminder date should be between current date time and scheduled date time",
+                "hcDefinedReminderText": "Enter the date and time of the reminder.",
+                "reminderalertTextDefined": "Reminder date should be between current date time and scheduled date time.",
```

Comment:
Replace with:
`Reminder date time should be between current and scheduled date time.`